nwb_conversion_tools/nwbconverter.py

```diff
@@ -97,7 +78,7 @@ def run_conversion(self, metadata_dict, nwbfile_path=None, save_to_file=True, st
 
         # Run data interfaces data conversion
         for name, data_interface in self.data_interface_objects.items():
-            data_interface.convert_data(nwbfile, metadata_dict[name], stub_test)
+            data_interface.convert_data(nwbfile, metadata_dict, stub_test)
```

Comment:
This is the main line that breaks previous converters; they could be easily refactored by adjusting to the new standard, though the majority of them are already uploaded to DANDI so I assume that isn't a priority.